spec/system/claims/sign_out_as_claims_user_spec.rb

```diff
@@ -3,13 +3,21 @@
 RSpec.describe "Sign out as a Claims User", type: :system, service: :claims do
   let(:colin) { create(:claims_support_user, :colin) }
 
+  before do
+    # rubocop:disable RSpec/AnyInstance
+    allow_any_instance_of(DfESignInUser).to receive(:logout_url).and_return("/auth/dfe/sign-out")
+    # rubocop:enable RSpec/AnyInstance
```

Comment:
Are we able to mock this directly onto something like `colin.dfe_sign_in_user`?

Reply:
Not exactly. We need to mock DFESignInUser and it's a bit of a pain

Reply:
Mainly because the `DFESignInUser` changes throughout this spec. If I mock it with an email address the spec will fail because it finds the user in DB and because of this he is logged in. If I mock it without an email, the user is basically logged out. 

The instance of `DFESignInUser` changes state and I don't think mocking the changed state is a good idea. So it feels much cleaner to just mock any instance of `DFESignInUser` to return the desired logout url